big_tests/tests/muc_light_SUITE.erl

```diff
@@ -792,6 +796,21 @@ remove_and_add_users(Config) ->
             assert_cache_hit_event(TS, 1, room_bin_jid(?ROOM))
         end).
 
+multiple_owner_change(Config) ->
```

Comment:
For me this is not the best way to test it, because:
- It unnecessarily changes Bob's and Kate's affiliations to `none`.
- It checks that there can be three owners, but I'd rather see a simple case, where you just change Bob to owner and see that both Alice and Bob are owners.
- Am I correct that there is no test that the `allow_multiple_owners` options actually changes anything? Because I'd like to see the same case, where Bob is changed to owner, and ensure that Alice loses ownership. Otherwise all tests would still pass even though the option might be always enabled.